src/main/java/bio/terra/workspace/service/job/JobService.java

```diff
@@ -52,6 +45,12 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.ScheduledFuture;
 import java.util.concurrent.TimeUnit;
+import org.apache.commons.lang3.StringUtils;
```

Comment:
I wonder if we can get the IntelliJ plugin and the spotless to agree on where the imports should go :(